content/influxdb3/core/reference/cli/influxdb3/serve.md

```diff
@@ -20,11 +20,13 @@ The `influxdb3 serve` command starts the {{< product-name >}} server.
 ```bash
 influxdb3 serve [OPTIONS] --node-id <HOST_IDENTIFIER_PREFIX>
 ```
+`--node-id` is a unique string that identifies your InfluxDB server instance
 
 ## Options
 
 | Option           |                                                      | Description                                                                                                               |
 | :--------------- | :--------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| {{< req "\*" >}} | `--node-id`                                          | _See [configuration options](/influxdb3/core/reference/config-options/#node-id)_                                          |
 |                  | `--object-store`                                     | _See [configuration options](/influxdb3/core/reference/config-options/#object-store)_                                     |
 |                  | `--bucket`                                           | _See [configuration options](/influxdb3/core/reference/config-options/#bucket)_                                           |
 |                  | `--data-dir`                                         | _See [configuration options](/influxdb3/core/reference/config-options/#data-dir)_                                         |
```

Comment:
Should `--object-store` and `--data-dir` also include the required asterisk?

Reply:
`object-store` defaults to `memory` (slated to change to `file`, I think, but anyway...) so maybe I should just note the default in the usage list. `data-dir` is only required if using `file`.